hack/docker-build.sh

```diff
@@ -27,6 +30,7 @@ fi
 docker-build() {
     name=$1
     docker build \
+      --platform ${GOOS}/${GOARCH} \
```

Comment:
May we need change docker version, whether it supports `--platform` ?

Reply:
all version, you can see https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope